raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -0,0 +1,217 @@
+import asyncio
+import time
+from dataclasses import dataclass
+
+import gevent
+import structlog
+from aiortc import RTCDataChannel, RTCPeerConnection, RTCSessionDescription
+from gevent import Greenlet
+from gevent.event import Event
+
+from raiden.network.transport.matrix.rtc.aiogevent import yield_future
+from raiden.network.transport.matrix.utils import my_place_or_yours
+from raiden.utils.formatting import to_checksum_address
+from raiden.utils.typing import Address, Any, Callable, Coroutine, Dict, List, Optional
+
+log = structlog.get_logger(__name__)
+
+
+@dataclass
+class RTCPartner:
+    partner_address: Address
+    peer_connection: RTCPeerConnection
+    channel: Optional[RTCDataChannel] = None
+    partner_ready_event: Event = Event()
```

Comment:
Does the default work? I thought one has to use `field(default[_factory]=...)` for defaults.

Reply:
I changed it, can you check it again?

Reply:
Looks good, same should be done for the `None` above.